### DISCUSSION TITLE
Endpoint Specific Limit

### DISCUSSION THREAD

**Discussion by YZmarkS** [rwf2/Rocket#2232] [Questions]
Hi Community, 

I would like to ask if there is a way to specify the upload limit on specific endpoints. 

I have one endpoint that is intended to receive just `.jar` files, which we expect to be rather small. Another endpoint that receives `.zip`, which we figure can be rather large. Is there a way to have different upload limit on the two endpoints? 

Thanks ahead of time

**Comment by the10thWiz** [comment]
Looking at the `TempFile` docs, it looks like `limits.file/$ext` is the configuration option to enable this. It doesn't allow differentiation by route, but for your use case it should be enough. (Note that the $ext is determined by the content_type you set on the route).

**Reply by YZmarkS** [reply]
Thanks for explanation. I will try this out.
